packages/web-api/src/WebClient.spec.js

```diff
@@ -115,6 +115,46 @@ describe('WebClient', function () {
     });
   });
 
+  describe('has an option to override the Axios timeout value', function () {
+    it('should log warning and throw error if timeout exceeded', function (done) {
+      const timeoutOverride = 15; // ms, guaranteed failure
+      
+      const logger = {
+        debug: sinon.spy(),
+        info: sinon.spy(),
+        warn: sinon.spy(),
+        error: sinon.spy(),
+        setLevel: sinon.spy(),
+        setName: sinon.spy(),
+      };
+      
+      const client = new WebClient(undefined, { 
+        timeout: timeoutOverride, 
+        retryConfig: { retries: 0 },
+        logLevel: LogLevel.WARN, 
+        logger 
+      });  
+      
+      client.apiCall('users.list')
+        .then(_ => {
+          done(new Error("expected timeout to throw error"));
+        })
+        .catch(error => {
+          try {
```

Comment:
Great job figuring out the nested try/catch to make the assert.equal failures actually display proper error messages!

Reply:
So happy we found a solution to that issue. Couldn't leave us in the dark with those!